client/src/App.test.js

```diff
@@ -1,37 +1,85 @@
 import React from 'react'
-import { render, screen } from '@testing-library/react'
-import { MemoryRouter, Router } from 'react-router-dom'
-import { createMemoryHistory } from 'history'
+import { render, screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { MemoryRouter, Redirect } from 'react-router-dom'
 import App from './App'
+import octocatRepos from './services/__mocks__/octocat-repos.json'
```

Comment:
Maybe also name it `octocatRepos.json` as well?
```suggestion
import octocatRepos from './services/__mocks__/octocatRepos.json'
```

Reply:
👍 consistency is important!